Copy to SD Card root directory to update/config.ini

```diff
@@ -55,12 +55,12 @@
 
 #### Enable Status Screen
 # Status screen or home screen displays the current temperature, fan and speeds.
-# If this disabled, the main menu will become the default home screen.
+# If this disabled, the main menu will become the default home screen
```

Comment:
Why remove periods in some places & not others? The line above this still has one.